### PR TITLE
Resolve issue with likes on single image page

### PR DESCRIPTION
**Pull request by Capitalsav**

### CODE REVIEW

**Comment by vladfreel** [comment]
if user_signed_in?
        UserAction.save_user_action(current_user.id, UserAction::ACTION_NAVIGATION, single_category_image_path)
      end

В before_action, а если точнее, все ACTION_NAVIGATION должны быть в ApplicationController в before_action

**Comment by vladfreel** [comment]
class ApplicationController < ActionController::Base
before_action :user_click
  private
    UserAction.save_user_action(current_user.id, UserAction::ACTION_NAVIGATION, request.original_url)

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -80,8 +80,7 @@ def show_images
 
   def show_one_image
     @image = Category.friendly.find(params[:id]).images.find(params[:image_id])
-    @likes_count = @image.likes.count
-    @comments = @image.comments
+    @like = @image.likes.where(user_id: current_user.id).first
```
Comment (drkmen) [inline app/controllers/categories_controller.rb]: @like = @image.likes.find_by(user_id: current_user.id)

File: app/views/categories/show_one_image.html.haml [hunk 2]
```diff
@@ -5,11 +5,6 @@
       .caption
       - if user_signed_in?
         = render "likes/like_form"
-      - if !user_signed_in?
```
Comment (drkmen) [inline app/views/categories/show_one_image.html.haml]: - unless user_signed_in?

**Comment by vladfreel** [comment]
Теперь вынеси UserAction.save_user_action в один метод в ApplicationController и вызывай его в тех местах со своими параметрами

**Pull request merged**